flake8_colors/formatter.py

```diff
@@ -32,10 +32,30 @@ class ColorFormatter(base.BaseFormatter):
     name = 'flake8-colors'
     version = '0.1.8'
 
+    @classmethod
+    def add_options(cls, options):
+        options.add_option(
+            '--color',
+            type='str',
+            default='auto',
+            choices=('default', 'always', 'never'),
+            help='when to colorize the output; can be "always", "auto" (default) or "never"',
+            parse_from_config=True,
+        )
+
     @classmethod
     def parse_options(cls, options):
-        # Only use color formatting if invoked interactively
-        if sys.__stdin__.isatty():
+        if options.color == 'always':
+            colorized = True
+        elif options.color == 'never':
+            colorized = False
+        else:  # options.color == 'default'
```

Comment:
Small typo here: should be 'auto', not 'default'

Reply:
Yes indeed, thanks !